_layouts/collection-list.html

```diff
@@ -0,0 +1,86 @@
+<!doctype html>
+
+{% include head.html %}
+
+<body>
+  {% include header-extended.html %}
+
+	<main id="main-content">
+	<section class="bg-primary-darker usa-section">
+	  <div class="grid-container margin-y-5">
+	    <div class="tablet:grid-col-9">
+	      <h1 class="text-base-lightest">Adobe Inc.</h1>
```

Comment:
We need to dig into this page content hierarchy a bit. Looks like this page title should be "Adobe Inc." Between that and the H2 and the given title ("Requirements"), something is amiss.

An indicator of the problem is that we have this content in the template instead of the content file. Should be pulling title and "body content" (if present) from the .md.